go-controller/pkg/clustermanager/userdefinednetwork/template/net-attach-def-template_test.go

```diff
@@ -15,10 +15,17 @@ import (
 )
 
 var _ = Describe("NetAttachDefTemplate", func() {
+
+	// before each test, set the IPv4Mode and IPv6Mode to true
+	BeforeEach(func() {
+		config.IPv4Mode = true
```

Comment:
why this is needed, I thought this is only used for default network. secondary network's ipmode are in its own secondaryNetInfo.

Reply:
without that, some test cases will fail with errors in below:

<go-controller/pkg/util/multi_network.go>
```
func newNetInfo(netconf *ovncnitypes.NetConf) (MutableNetInfo, error) {

	if ni.IsPrimaryNetwork() && ni.IsSecondary() {
		ipv4Mode, ipv6Mode := ni.IPMode()
		if ipv4Mode && !config.IPv4Mode {
			return nil, fmt.Errorf("network %s is attempting to use ipv4 subnets but the cluster does not support ipv4", ni.GetNetworkName())
		}
		if ipv6Mode && !config.IPv6Mode {
			return nil, fmt.Errorf("network %s is attempting to use ipv6 subnets but the cluster does not support ipv6", ni.GetNetworkName())
		}
	}
	return ni, nil
}
```

Reply:
I see, didn't know that secondary network also replies on the default ipfamily config.

lgtm then.